font-codegen/src/fields.rs

```diff
@@ -475,10 +475,15 @@ impl Field {
     }
 
     pub(crate) fn is_zerocopy_compatible(&self) -> bool {
-        matches!(
-            self.typ,
-            FieldType::Scalar { .. } | FieldType::Offset { .. }
-        )
+        // hack: we want to add `FieldType::Struct` here but don't want to
+        // catch `ValueRecord` so use this attribute to ignore it.
+        // Fields that require args for reading can't be read "zerocopy"
```

Comment:
I'm not quite sure what meaning to take from this. Do you mean anything with a variable number of something has to be copied? - I didn't think that was true.

Reply:
It means if we enable this for ValueRecord it breaks everything. I’ll add an issue for addressing codegen hacks around embedded records but I’ll have to defer to @cmyr about what the best approach might be.

Reply:
It means if we enable this for ValueRecord it breaks everything. I’ll add an issue for addressing codegen hacks around embedded records but I’ll have to defer to @cmyr about what the best approach might be.

Reply:
To clarify there are more issues than just `ValueRecord`; the issue is *any* record that is not fixed size, of which there are about ten (including  things like `PairValueRecord` `Tuple`, `VariationRegion`, `Mark2Record`, etc).

I also think that this change is incorrect, since a struct can only be included as a field of another record *if* it is copy, which this does not guarantee.